releasetool/__main__.py

```diff
@@ -165,10 +176,31 @@ def publish_reporter_start(github_token: str, pr: str):
 @click.option("--pr", envvar="AUTORELEASE_PR", default=None)
 @click.option("--status", type=bool, default=True)
 @click.option("--details", envvar="PUBLISH_DETAILS", default=None)
-def publish_reporter_finish(github_token: str, pr: str, status: bool, details: str):
+@click.option("--app_id_path", envvar="APP_ID_PATH", default=None)
+@click.option("--installation_path", envvar="INSTALLATION_PATH", default=None)
+@click.option("--private_key_path", envvar="GITHUB_PRIVATE_KEY_PATH", default=None)
+def publish_reporter_finish(
+    github_token: str,
+    pr: str,
+    status: bool,
+    details: str,
+    app_id_path: str,
+    installation_path: str,
+    private_key_path: str,
+):
+    if app_id_path:
+        github_token = github_jwt_dict(app_id_path, installation_path, private_key_path)
     releasetool.commands.publish_reporter.finish(github_token, pr, status, details)
 
 
+def github_jwt_dict(app_id_path: str, installation_path: str, private_key_path: str):
+    return {
```

Comment:
Add a """doc comment""".    The Visual Studio Code extension "Python Docstring Generator" will do most of the work.